vue-form.js

```diff
@@ -460,8 +465,8 @@
                 this._vueFormCtrl.validate(value);                
             },
             unbind: function () {
-                this._vueForm.removeControl(this._vueFormCtrl);
-                delete this.el._vueFormCtrl;
+                if(this._vueForm) this._vueForm.removeControl(this._vueFormCtrl);
+                if(this.el) delete this.el._vueFormCtrl;
```

Comment:
the form is in a v-if section, when toggle, it raise a error here, these if statement fix this kind of error, just a quick fix, not sure why this happen. may need more test.